doc/user_doc/vic_app_dev/using_volumes_with_vic.md

```diff
@@ -0,0 +1,75 @@
+# Using Volumes with vSphere Integrated Containers #
+
+vSphere Integrated Containers supports the use of container volumes. When you create or the vSphere Administrator creates a virtual container host, you or the Administrator specify the datastore to use to store container volumes in the `vic-machine create --volume-store` option. For information about how to use the `vic-machine create --volume-store` option, see the section on `volume-store` in [Virtual Container Host Deployment Options](../vic_installation/vch_installer_options.html#volume-store) in *vSphere Integrated Containers Installation and Configuration*.   
+
+## Obtain the List of Available Volume Stores ##
+
+To obtain the list of volume stores that are available on a virtual container host, run `docker info`.
+
+<pre>docker -H <i>virtual_container_host_address</i>:2376 --tls info</pre>
+
+The list of available volume stores for this virtual container host appears in the `docker info` output under `VolumeStores`.
+
+<pre>[...]
+Storage Driver: vSphere Integrated Containers Backend Engine
+VolumeStores: <i>volume_store_1</i> <i>volume_store_2</i> ... <i>volume_store_n</i>
+vSphere Integrated Containers Backend Engine: RUNNING
+[...]</pre>
+
+## Create a Volume in a Volume Store ##
```

Comment:
How's this?

- If you intend to create anonymous volumes by using `docker create -v`, a volume store named `default` must exist. In this case, you include the path to the source from which to mount an anonymous volume in the `docker create -v` command. Docker creates the volume in the `default` volume store, if it exists.

  <pre>docker -H <i>virtual_container_host_address</i>:2376 --tls create 
-v /var/myApp/data busybox</pre>

  **NOTE**: If you use `docker create -v`, vSphere Integrated Containers only supports the `-r` and `-rw` options.